### PR TITLE
Adding fields removed from SourceCardOptions in merge

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
r? @remi-stripe 
cc @stripe/api-libraries 

When I merged SourceCard with CreditCard options some of this was dropped, then picked back up by a bug fix.

### CODE REVIEW

File: src/Stripe.net/Services/Sources/SourceCardOptions.cs [hunk 1]
```diff
@@ -25,6 +26,9 @@ public class SourceCardOptions : INestedOptions
         [JsonProperty("currency")]
         public string Currency { get; set; }
 
+        [JsonProperty("default_for_currency")]
+        public bool? DefaultForCurrency { get; set; }
+
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Sources/SourceCardOptions.cs:31]: Why is this here? `default_for_currency` is for *external accounts* so it doesn't match a class named `SourceCardOptions`

Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Sources/SourceCardOptions.cs:31]: Ah, added that as an override because it was showing as removed from something but I can't find which so will remove.

Comment (cjavilla-stripe) [reply src/Stripe.net/Services/Sources/SourceCardOptions.cs]: Removed
